src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -16465,6 +16465,49 @@ public class Derived : Base
         }
 
         [Fact]
```

Comment:
Can't click on restore.cmd so clicking here.

It says the file mode changed.  That shouldn't be necessary for the PR.  Can you please revert this?